tests/CMakeLists.txt

```diff
@@ -76,6 +79,16 @@ if(HAVE_BFD_DISASM)
   target_link_libraries(bpftrace_test ${LIBOPCODES_LIBRARIES})
 endif(HAVE_BFD_DISASM)
 
+target_compile_definitions(bpftrace_test PRIVATE HAVE_OVERRIDE_RETURN)
+if(NOT HAVE_OVERRIDE_RETURN)
+  set_property(
+    SOURCE ${CMAKE_SOURCE_DIR}/src/ast/irbuilderbpf.cpp
+    APPEND
+    PROPERTY COMPILE_DEFINITIONS
+    BPF_FUNC_override_return=58
```

Comment:
Could you add a comment explaining what the meaning of 58 is please? (even if it's just an arbitrary value, it'll make people less scared of changing this code in the future)